src/cheri_prelude_128.sail

```diff
@@ -75,10 +77,10 @@ struct Capability = {
   internal_e             : bool    ,
   E                      : bits(6) ,
   sealed                 : bool    ,
```

Comment:
Do we still need the sealed boolean?

Reply:
When I did the mips version of this change I kept it for convenience (quite a few places refer to cap.sealed) and kept it synchronised with type field, although it is probably more appropriate to replace it with an isSealed function.